Enterprise-CMCS/macpro-mdct-qmr
# Additional Fields in LSC-CH & CPU-AD

### Description


MDCT-2617
In LSC-CH, Measurement Specification initially choosing “NCQA/HEDIS” option, the dropdown and Performance Measure section do not dynamically appear. There were extra files made that weren't needed causing issue in LSC-CH, does not affect other measures.

MDCT-2618
Updates to LSC-CH, the Performance Measure section was showing the same fields no matter if the Measurement Specification radio was NCQA/HEDIS or Other. The original LSC-CH was using the wrong performance measure component, swapping it fixed most issues. Also removed inaccurate data source.

MDCT-2623
Both LSC-CH & CPU-AD needed to capture measure-eligible population and sample size. A copy was made with the header removed.


### Related ticket(s)


MDCT-2617
MDCT-2623
MDCT-2618

---
### How to test

1) Sign into QMR, any account
2) Select Reporting Year 2023
3) Go to LSC-CH measure,

For MDCT-2618
4) Compare LSC-CH to **LSC-CH_toCMS_updated 4.6.2023** attached to MDCT-2618, specifically the sections for Performance Measure.

For MDCT-2617
5) Measure Specification: Hedis selection should not have any issues displaying when selecting NCQA/HEDIS first

For MDCT-2623
6) For both LSC-CH & CPU-AD, under **Definition of Population Included in the Measure** section, these fields should be there
<img width="728" alt="Screenshot 2023-06-14 at 3 31 27 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/127151429/3a9610cf-cbff-462f-9aa9-88c7825aaec5">


### Important updates


N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
